Zend/zend_compile.c

```diff
@@ -3862,6 +3863,22 @@ int zend_compile_func_get_args(znode *result, zend_ast_list *args) /* {{{ */
 }
 /* }}} */
 
+int zend_compile_func_array_key_exists(znode *result, zend_ast_list *args) /* {{{ */
+{
+	if (args->children != 2 || args->child[0]->kind == ZEND_AST_UNPACK) {
```

Comment:
This also needs to check whether args->child[1] is an unpack.

Reply:
done